src/Maestrano/Configuration/WebhookConnec.cs

```diff
@@ -6,34 +6,28 @@ namespace Maestrano.Configuration
 {
     public class WebhookConnec
     {
-        public WebhookConnecSubscriptions Subscriptions { get; set; }
-
         /// <summary>
         /// Load Webhook into a WebhookAccount configuration object from a JObject 
         /// </summary>
         /// <returns>A WebhookAccount configuration object</returns>
         public static WebhookConnec LoadFromJson(JObject obj)
         {
             var config = new WebhookConnec();
-            config.NotificationsPath = obj["notification_path"].Value<string>();
-            var subscriptions = obj["connec_subscriptions"];
-            if (subscriptions != null)
-            {
-                config.Subscriptions = WebhookConnecSubscriptions.LoadFromJson(subscriptions.Value<JObject>());
-            }
-            else
-            {
-                config.Subscriptions = new WebhookConnecSubscriptions();
-            }
-
-            
+            config.NotificationPath = obj["notification_path"].Value<string>();
+            config.InitializationPath = obj["initialization_path"].Value<string>();
+            config.ExternalIds = obj["external_ids"].Value<bool>();
             return config;
         }
 
         /// <summary>
+        /// return true if connec will use external ids
+        /// </summary>
+        public bool ExternalIds { get; set; }
+
+        /// <summary>
         /// Whether to receive notifications related to the customer company
         /// </summary>
-       public string NotificationsPath { get; set; }
+        public string NotificationPath { get; set; }
```

Comment:
I would keep a deprecated getter NotificationsPath for backward compatibility purpose.